src/gui/scheduling/etagwatcher.cpp

```diff
@@ -81,6 +83,7 @@ void ETagWatcher::updateEtag(Folder *f, const QString &etag)
     if (OC_ENSURE_NOT(etag.isEmpty())) {
         auto &info = _lastEtagJob[f];
         if (f->canSync() && info.etag != etag) {
+            qCDebug(lcEtagWatcher) << "Scheduling etag update for space" << f->displayName();
```

Comment:
qCDebug(lcEtagWatcher) << "Scheduling sync of" << f->displayName() << "due to an etag change";

Reply:
done